src/README.md

```diff
@@ -231,7 +231,7 @@ custom_streamer = CustomStreamer()
 
 pipe.generate("The Sun is yellow because", max_new_tokens=15, streamer=custom_streamer)
 ```
-For fully implemented iterable CustomStreamer please refer to [multinomial_causal_lm](https://github.com/openvinotoolkit/openvino.genai/tree/releases/2024/3/samples/python/multinomial_causal_lm/README.md) sample.
+For fully implemented iterable CustomStreamer please refer to [multinomial_causal_lm](https://github.com/openvinotoolkit/openvino.genai/tree/releases/2024/3/samples/python/text_generation/README.md) sample.
```

Comment:
I have also found that OpenVINO docs refer to GenAI samples locations

![{9043CD71-49CD-4EA1-9EB9-649F69350A67}](https://github.com/user-attachments/assets/179cff73-5884-465a-bed1-09857148398b)

Please, prepare PRs to OpenVINO repo as well:
- releases/2024/6 branch to point to GenAI's releases/2024/6
- master branch to point to GenAI's master with updated locations.

Reply:
looks like there is no changed paths in openvino/docs/articles_en/learn-openvino/llm_inference_guide/genai-guide.rst on master branch

Reply:
https://github.com/openvinotoolkit/openvino/pull/28384 - master
https://github.com/openvinotoolkit/openvino/pull/28383 - 2024/6